### PR TITLE
Fix undefined array key

### PR DESCRIPTION
**Pull request by dzatoah**
Fixes: Undefined array key "all" (/usr/share/gosa/plugins/personal/privacyidea/tokens/class_MFATokenHelper.inc, line 136)

### CODE REVIEW

File: personal/privacyidea/tokens/class_MFATokenHelper.inc [hunk 1]
```diff
@@ -133,6 +133,8 @@ class MFATokenHelper
         /* START COMPARISONS */
         // Skip comparison if token limit is 0. That means there is no maximum limit for tokens.
         if ($tokenLimitAll != 0) {
+            $this->utils->tokenCounts["all"] = $this->utils->tokenCounts["all"] ?? 0;
```
Comment (sunweaver) [inline personal/privacyidea/tokens/class_MFATokenHelper.inc:136]: Does this also cover the case where the array $this->utils->tokenCounts does not have the element tokenCounts["all"] set?

**Review by sunweaver — APPROVED** [review]
ACK.

**Comment by sunweaver** [comment]
@dzatoah This PR's update branch needs a manual rebase. Thanks!

File: personal/privacyidea/tokens/class_MFATokenHelper.inc (hunk 1 above)
Comment (dzatoah) [reply personal/privacyidea/tokens/class_MFATokenHelper.inc:150]: This is exactly what this line does. If its not set, then set it to 0.

**Comment by dzatoah** [comment]
Rebased @sunweaver r4merge

**Review by sunweaver — APPROVED** [review]
LGTM now.

**Pull request merged**